### ISSUE TITLE
About the loss calculation in audio.

### ISSUE THREAD

**Issue by musicrainie** [AryanShekarlaban/data2vec-pytorch#22]
Maybe this line
https://github.com/arxyzan/data2vec-pytorch/blob/61c0aa8d500c4101f8626b080bfe2b6263f5695e/audio/trainer.py#L61
should be fixed to 
`loss = self.criterion(x.float(), y.float()).sum(dim=-1).sum().div(x.size(0))`
to convert MATRIX to SCALAR, just like the text and vision case:
https://github.com/arxyzan/data2vec-pytorch/blob/61c0aa8d500c4101f8626b080bfe2b6263f5695e/text/trainer.py#L69
https://github.com/arxyzan/data2vec-pytorch/blob/61c0aa8d500c4101f8626b080bfe2b6263f5695e/vision/trainer.py#L57
.

**Comment by arxyzan** [comment]
Hello @musicrainie sorry for the delayed reply. Unfortunately I don't really remember much of this code any more but as far as I remember, I tried to carefully replicate the original behavior. But I can be mistaken.
